bc_events/client.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, api_url, service_name, topic_definitions):
 
         self.api_url = api_url
         self.publish_url = api_url + "/events" if api_url else None
+        self.publish_all_url = self.publish_url + "/bulk/" if api_url else None
```

Comment:
Last cleanup item. Just for consistency sake, let's make this `publish_bulk_url` to match the `publish_bulk` method it is used in.